### ISSUE TITLE
Doc: How to use LaTeX -> myst

### ISSUE THREAD

**Issue by LecrisUT** [executablebooks/mystmd#1007]
## Proposal

There were some mentions of LaTeX -> myst and there is the equivalent package, but it is unclear how one would use it. Is it supposed to be an equivalent source like an `.md` file? Some documentation would be helpful

**Comment by rowanc1** [comment]
Any latex source files that are in the project will be picked up and transformed. There is a [blog post here](https://curvenote.com/blog/how-to-use-latex-with-myst-markdown) and  some examples in [this repository](https://github.com/physiome-journal) that we use for [Physiome](https://journal.physiomeproject.org/).

Docs would be wonderful for this portion of the workflow and happy for contributions to start it! (I think that the structure of the docs may need to be adapted, but probably just a new page that talks about this a bit would be a great addition, and we can organize later).

**Comment by rowanc1** [comment]
The first cut at this is here:
https://mystmd.org/guide/writing-in-latex

Happy for feedback!

**Comment by LecrisUT** [comment]
From what I read there, there is no frontmatter section for LaTeX?

Maybe some other undocumented limitations would be splitting the document in multiple files? That would be a limitation for book projects.

**Comment by rowanc1** [comment]
I will take another pass on documenting how the frontmatter is dealt with (i.e. everything before `\begin{document}`), which is parsed, but there are some limitations / quirks.

It is also possible to use snippets of latex (e.g. a latex table):

https://mystmd.org/guide/embed#docs-include

![image](https://github.com/executablebooks/mystmd/assets/913249/289a7b69-4931-43c9-9b5c-be87d4e75ae5)

You can split up files and use `include` in latex as well, but you might have a different thing in mind for book projects?

**Comment by LecrisUT** [comment]
> You can split up files and use `include` in latex as well, but you might have a different thing in mind for book projects?

Would `mystmd` try to parse any `.tex` file and create a page for it or only those that have `\begin{document}`? If the former, that would create a problem for book projects where the content of included chapters do no have `\begin{document}`. That is the main problem I was thinking of

**Comment by LecrisUT** [comment]
> The way that web is presented is often different, for example, in [my thesis](https://phd.row1.ca/framework), each chapter is a page online, but there is only one `\begin{document}`.

Just to clarify that is independent of the file separation and it's governed by the `frontmatter` right?

**Comment by rowanc1** [comment]
Correct, the configuration is in the `_toc.yml` at the moment ([docs](https://mystmd.org/guide/table-of-contents)) and does not depend on the file separation choices.
